doc/api/modules.md

```diff
@@ -377,9 +391,11 @@ If the exact filename is not found, then Node.js will attempt to load the
 required filename with the added extensions: `.js`, `.json`, and finally
 `.node`.
 
-`.js` files are interpreted as JavaScript text files, and `.json` files are
-parsed as JSON text files. `.node` files are interpreted as compiled addon
-modules loaded with `process.dlopen()`.
+`.json` files are parsed as JSON text files, `.node` files are interpreted as
+compiled addon modules loaded with `process.dlopen()`. Files using any other
+extension (or no extension at all) are parsed as JavaScript text files. Refer to
+the [Determining module system][] section to understand what parse goal will be
+used.
 
```

Comment:
```suggestion

`.cjs` files always need their extension to be included:
`require('./file.cjs')`. The `.cjs` extension cannot be left out
of `require` calls like `.js` can be.
```

Reply:
I've added a note about this in 5ed7e1cdf3d36a4e396e6cceade02743f872b4ea, let me know if that works for you.

Reply:
Sure, looks good.